src/store.ts

```diff
@@ -1,9 +1,8 @@
 import { INITIAL_STATE_KEY } from './keys';
-import { Action } from '@ngrx/store';
 
-export function Store<TState>(initialState?: TState): (target: Function) => void;
-export function Store(initialState?: any): (target: Function) => void;
-export function Store(initialState: any = {}) {
+export function Store<TState extends object>(initialState?: TState): (target: Function) => void;
```

Comment:
I don't think that this is a good idea. A feature state can also be a number. Can we go back to `any`?

Reply:
I did not know that